File: src/targets/python/requests/fixtures/multipart-data.py
import requests

url = "https://httpbin.org/anything"

files = { "foo": ("src/fixtures/files/hello.txt", open("src/fixtures/files/hello.txt", "rb"), "text/plain") }
payload = { "bar": "Bonjour le monde" }

response = requests.post(url, data=payload, files=files)

print(response.text)

File: src/targets/csharp/httpclient/fixtures/multipart-data.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("https://httpbin.org/anything"),
    Content = new MultipartFormDataContent
    {
        new StringContent("Hello World")
        {
            Headers =
            {
                ContentType = new MediaTypeHeaderValue("text/plain"),
                ContentDisposition = new ContentDispositionHeaderValue("form-data")
                {
                    Name = "foo",
                    FileName = "src/fixtures/files/hello.txt",
                }
            }
        },
        new StringContent("Bonjour le monde")
        {
            Headers =
            {
                ContentDisposition = new ContentDispositionHeaderValue("form-data")
                {
                    Name = "bar",
                }
            }
        },
    },
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}